packages/emotion-theming/__tests__/with-theme.js

```diff
@@ -30,6 +30,7 @@ test(`withTheme(Comp) hoists non-react static class properties`, () => {
   const ComponentWithTheme = withTheme(ExampleComponent)
 
   expect(ComponentWithTheme.displayName).toBe('WithTheme(foo)')
+  // $FlowFixMe
```

Comment:
it would have made sense to add an explanation of why this is needed, for future reference.

I guess it's too late.

Reply:
'tis never too late! Made a new PR https://github.com/emotion-js/emotion/pull/1399